config.toml.sample

```diff
@@ -9,7 +9,7 @@ template = "super" # Results template
 rules_json = "/etc/super/rules.json" # Vulnerability rules JSON
 
 # Vulnerable or potentially vulnerable permissions
-[[permissions]]
+[unknown_permissions]
```

Comment:
This is the unique way I've found to be able to load the unknown permissions, but this can break previous configs!